browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2135,7 +2135,7 @@ Are you sure you want to do this?
       You can create a secondary account by importing your private key.
     </message>
     <message name="IDS_IMPORT_ACCOUNT_TEXT" desc="Brave wallet accounts text">
-      Enter, paste, or import your private key string file or JSON.
+      Import from a JSON file only.
```

Comment:
that's not right as private key import works. How do you suppose to import from a private key?

Reply:
gonna study the behaviour from all platforms and will try to reproduce the failed case then will update the PR and current UI accordingly.